blivet/devicelibs/stratis.py

```
@@ -308,10 +308,10 @@ def add_device(pool_uuid, device):
         proxy = util.SystemBus.get_proxy(STRATIS_SERVICE, pool_info.object_path, STRATIS_POOL_INTF)
         ((succ, _paths), rc, err) = proxy.AddDataDevs([device], timeout=STRATIS_CALL_TIMEOUT)
     except DBusError as e:
-        raise StratisError("Failed to create stratis filesystem on '%s': %s" % (pool_info.name, str(e)))
+        raise StratisError("Failed to add device '%s' to stratis pool '%s': %s" % (device, pool_info.name, str(e)))
```

Ruff (0.15.18)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)